test/HealthChecks.Azure.IoTHub.Tests/HealthChecks.Azure.IoTHub.Tests.csproj

```
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.10.5" />
```

in the meantime we have merged CPM, so providing a version is not needed here

```suggestion
    <PackageReference Include="Azure.Identity" />
```